.github/CODEOWNERS

```diff
@@ -743,6 +743,9 @@
 # ServiceLabel: %Notification Hub
 # ServiceOwners:                                                   @tjsomasundaram
 
+# PRLabel: %Network - DNS
+/sdk/dnsresolver/                                                  @jamesvoongms @jotrivet @harmen-msft @anilpuvvadi @arpit-gagneja
```

Comment:
This needs to move to the management section (~L963) and follow the patterns there there.   This also needs a ServiceOweners block so that the right folks are pulled in for support.

The accounts that you're using must all be properly onboarded to the Azure SDK repository.   Looks like none of them are.

Reply:
Fixed, only added user who is onboarded